### PR TITLE
Improve responsive columns behavior 

### PR DESCRIPTION
**Pull request by Volker-E**
Bug: [T193920](https://phabricator.wikimedia.org/T193920)

### CODE REVIEW

File: css/wmui-style-guide.dev.css [hunk 1]
```diff
@@ -789,15 +792,17 @@ sup {
...
 
 	.col--start {
-		width: 25%;
+		width: 24rem;
 	}
+
 	.col--end {
-		width: 75%;
+		margin-left: -24rem;
+		padding-left: 27.2rem; /* = 24 + (2 * 1.6) */
```
Comment (prtksxna) [inline css/wmui-style-guide.dev.css]: Use the same comment as L778?

File: css/wmui-style-guide.dev.css [hunk 2]
```diff
@@ -816,4 +821,13 @@ sup {
...
 		width: var( --width-site-desktop-wide );
 	}
+
+	.col--start {
+		width: 25.6rem;
+	}
+
+	.col--end {
+		margin-left: -25.6rem;
+		padding-left: 28.8rem; /* = 25.6 + 1.6 */
```
Comment (prtksxna) [inline css/wmui-style-guide.dev.css]: Should this also be `+1.6+1.6`?

File: css/wmui-style-guide.dev.css (hunk 1 above)
Comment (Volker-E) [reply css/wmui-style-guide.dev.css]: Done and clarified a bit more.

File: css/wmui-style-guide.dev.css (hunk 2 above)
Comment (Volker-E) [reply css/wmui-style-guide.dev.css]: Done and clarified a bit more.

**Pull request merged**